masterlist.yaml

```
@@ -23577,6 +23577,108 @@ plugins:
       - <<: *alreadyInOrFixedByX
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp")'
+    
+    ### Dawn of Skyrim - Modern Patches ###
```

There's some extra whitespace, & the subsection header should only be indented once:

```suggestion

  ### Dawn of Skyrim - Modern Patches ###
```